bagh/bagh.json

```diff
@@ -5,6 +5,14 @@
   "status": "beschikbaar",
   "version": "0.0.1",
   "crs": "EPSG:28992",
+  "versioning": {
```

Comment:
Versioning dekt de lading niet helemaal en wordt ook verwarrend als we ook schema versies mee gaan nemen. Dit gaat meer temporaliteit dus ik zou dit dus juist `temporal` noemen.